### PR TITLE
[PLAYER-4483] Fixed app crash when removed playback speed options from skin.json

### PR DESCRIPTION
**Pull request by MarkEPAM**
- fixed app crash when removed playback speed options from skin.json

Unit tests not included.
Ticket: https://jira.corp.ooyala.com:8443/browse/PLAYER-4483

### CODE REVIEW

**Review by akshaydch11 — APPROVED** [review]

**Comment by akshaydch11** [comment]
LGTM

File: sdk/react/ooyalaSkinPanelRenderer.js [hunk 1]
```diff
@@ -220,9 +220,16 @@ OoyalaSkinPanelRenderer.prototype.renderAudioAndCCSelectionPanel = function() {
 };
 
 OoyalaSkinPanelRenderer.prototype.renderPlaybackSpeedPanel = function() {
-  return (
+  let playbackSpeedRates;
+
+  if (this.skin.props.playbackSpeed && Array.isArray(this.skin.props.playbackSpeed.options)) {
+    playbackSpeedRates = this.skin.props.playbackSpeed.options;
+  } else {
+    playbackSpeedRates = [];
```
Comment (brol1dev) [inline sdk/react/ooyalaSkinPanelRenderer.js]: you can simplify this logic. The else can be omitted if you initialize the variable as an empty array. E.g.
```
let playbackSpeedRates = [];
if (this.skin.props.playbackSpeed && Array.isArray(this.skin.props.playbackSpeed.options)) {
    playbackSpeedRates = this.skin.props.playbackSpeed.options;
  }
```

**Review by brol1dev — CHANGES_REQUESTED** [review]
requested a change. If EPAM doesn't do it today because of their time zone I'll add the change and merge it so we can test it.

**Review by brol1dev — APPROVED** [review]
Made the change myself, you can check it out @MarkEPAM

**Pull request merged**